src/topic/BecomeMemberContainer.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import TopicModel from '../shared/models/TopicModel';
+import { becomeMember, leaveTopic } from '../actions/member.actions';
+import { UserModel } from '../shared/models/UserModel';
+
+class BecomeMemberContainer extends React.Component {
+    render() {
+        const { topic, user, attend, leave } = this.props;
+        const alreadyAttending = topic.amIAttending(user);
+        return (
+            <div>
```

Comment:
we can't accept divs, that's why we have `styled-components`

Reply:
Could you please shoot a few point why? afair `styled.component` looks like "styled.div`....`" which is still a div (i.e. both have the same result)

Reply:
changed in https://github.com/warsawjs/topics-manager/pull/27/commits/6232648170503c80518ff8059c6d3be5975f81a1

Reply:
each `div` is an indicator we need a styledComponent, 
we don't want to use divs, because accessibility matters, see http://www.a11ymatters.com/article/dont-use-divs-for-everything/
You can also use Fragments https://reactjs.org/docs/fragments.html if there are no styles needed